features/foutafhandeling.feature

```diff
@@ -290,9 +290,21 @@ Rule: min waarde is niet groter dan max waarde
         | code   | paramsValidation                              |
         | detail | Bad request.                                  |
         En bevat de response de volgende invalidParams
-        | name             | code   | reason                           |
-        | oppervlakte[min] | range  | min mag niet hoger zijn dan max. |
+        | name             | code  | reason                           |
+        | oppervlakte[min] | range | min mag niet hoger zijn dan max. |
 
         Voorbeelden:
         | path                   | query string                               | code  | reason                           |
         | /adresseerbareobjecten | ?oppervlakte[min]=200&oppervlakte[max]=100 | range | min mag niet hoger zijn dan max. |
+
+Rule: bij geometrie parameters hebben waarde fouten voorrang op crs fouten
```

Comment:
is dit echt een rule? eerder een beperking in de implementatie. naar mijn mening een bug. wanneer er meerdere fouten zijn moeten die allemaal tegelijk worden gemeld. Dat gebeurt nu niet, zie #503.